test/form-builder/CheckboxFieldGroup.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+
+import CheckboxFieldGroup from '../../src/form-builder/CheckboxFieldGroup';
+import VaCheckboxGroup from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { buildRenderForm, changeValue } from '../utils';
+
+const renderForm = buildRenderForm({ thing: false, breakfast: [] });
+
+const getCheckboxGroupContainer = (container: HTMLElement): HTMLElement => {
+  const checkboxGroupContainer = container.querySelector(
+    'va-checkbox-group'
+  ) as HTMLElement;
+  if (!checkboxGroupContainer) throw new Error('No va-checkbox found');
+  return checkboxGroupContainer;
+};
+
+const testData = {
+  label: 'What breakfast?',
+  name: 'breakfast',
+  id: '12',
+  /**
+   * If `required` is true, the default message will be used. If `required` is a
+   * string, it will be used as the error message.
+   */
+  required: true,
+  values: {},
+  options: [
+    {
+      name: 'eggs',
+      label: 'Eggs',
+      content: '🐥🐣',
+      required: false,
+    },
+    {
+      name: 'protein',
+      label: 'Protein Shake',
+      content: '🏋️',
+      required: true,
+    },
+    {
+      name: 'toast',
+      label: 'Toast',
+      content: '🍞',
+      required: false,
+    },
+    {
+      name: 'fruit',
+      label: 'Fruit',
+      content: '🍐',
+      required: false,
+    },
+  ],
+};
+
+describe('Form Builder - CheckboxFieldGroup', () => {
```

Comment:
There are instances in some va forms where we provide a list of checkbox options, but don't require a user to select any, so the question is optional. Can you add a test that verifies no error message is thrown when the `required` prop is false? Thanks!